nextjs_tdd/src/app/@airlines/page.tsx

```diff
@@ -0,0 +1,5 @@
+import { COMMON_CONST } from '@/constants/app-constants'
+
+export default function AirlinesPage() {
+    return <div>{COMMON_CONST.CHOOSE_TO_FLY}</div>
```

Comment:
Having text in div is acceptable, but try to use semantic HTML elements whenever possible to improve the accessibility and readability of your code. 
eg., <p> or <h1> to <h6> or <span>

Reply:
Used HTML elements.